tests/test_parser.py

```diff
@@ -3,10 +3,18 @@
 
 class TestGraphParser:
 
-    def test_dummy(self):
-        graph_parser = GraphParser({})
+    def test_graph_parser(self):
+        graph_parser = GraphParser({'title': 'text', 'name': 'person_entity'})
         assert 'parse' in dir(graph_parser)
-
+        graph = graph_parser.parse('./data/test_data/data.json')
```

Comment:
- As mentioned in one of the previous comments, this test data should be generated **during** this test and the file should be **removed** after testing.
- Always use `assert` for testing instead of `print`.
- You might consider writing another test using different data should certain cases not covered in this one.